app/src/main/res/values-it/help.xml

```diff
@@ -51,32 +51,32 @@
     <string name="help_details"><![CDATA[
         <h1>Dettagli del Portafoglio</h1>
         <h2>Indirizzo pubblico</h2>
-		Il tuo indirizzo pubblico Monero equivale al tuo IBAN. Puoi condividerlo con chiunque senza temere di perdere i tuoi Moneroj. Gli altri invieranno Moneroj al tuo portafoglio utilizzando questo indirizzo. 
+		Il tuo indirizzo pubblico Monero equivale al tuo IBAN. Puoi condividerlo con chiunque senza temere di perdere i tuoi Moneroj. I Moneroj verranno inviati al tuo portafoglio utilizzando questo indirizzo. 
         <h2>Seed Mnemonico</h2>
         Questa è l&apos;unica informazione necessaria per recuperare il tuo portafoglio in futuro
 		e avere pieno accesso ai tuoi fondi.
 		E&apos; molto importante mantenerlo privato e conservarlo in un luogo sicuro, poiché può concedere a <em>chiunque</em>
 		pieno accesso ai tuoi Moneroj! Se non lo hai già fatto, annotati il Seed Mnemonico e conservalo in un luogo sicuro!
-        <h2>Wallet Files Recovery Password</h2>
-        Make sure you write this password down. If you reset your device or uninstall the app, you
-        will need it to access your wallet again.<br/>
+        <h2>Password di recupero files del portafoglio</h2>
```

Comment:
This heading should match the string ```generate_crazypass_label``` in ```strings.xml``` because that's what this is explaining. Thanks!

(A this is a UI label: The shorter the better)

Reply:
Edited. Yes this string is a bit long but it's really hard to keep them short in italian, we use a lot of words :)

Reply:
heh - i noticed that - that's probably why you speak so fast!